tests/integration/test_healthchecks.py

```diff
@@ -152,6 +133,7 @@ def store_internal_error_event():
         try:
             # These events will consume all the disk sapce and we will report not ready.
             relay.send_event(project_key)
+            relay.send_event(project_key)
```

Comment:
Why did this test change?

Reply:
To unflaky it , but it was already done in the separate PR, after merging `master` this change will disappear.